PermissionsEx/permissions.yml

```diff
@@ -595,6 +595,30 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
+ terminatorgenisys:
```

Comment:
spaced incorrectly.
is there a picture of the terminator genisys model? i can't find one on google